frontend/src/app/pages/Data/Layout.tsx

```diff
@@ -26,14 +26,14 @@ const Radio = ({ name, options }) => {
     const { actions } = useNationalMapSlice();
 
```

Comment:
The original way this Radio component was set up used the enum id to set the id and htmlFor value. This meant that when two Radio components were on the page, clicking on the label in the second set of Radio buttons changed the radio values on the first set. Adding a data attribute for the object id and creating a more specific id/htmlFor fixes that issue.